### ISSUE TITLE
RG3205W Rev5

### ISSUE THREAD

**Issue by Thecloudff7** [zry98/movistar-home-hacks#2]
Buenas a todos,

Adquirí un movistar home como asistente en casa, pero al ver que tenia pocas funcionalidades quería darle otro uso.

Vi este post (que por cierto muy bien explicado!)

Al desmontarlo, he visto que la placa base no se corresponde a la de la explicación. Por lo que veo la placa base es la rev5.

Estoy entrando ahora en tema de electronica, y la verdad que no me aclaro. ¿Podríais decirme donde tendría que soldar el usb?

![Imagen de WhatsApp 2022-11-04 a las 20 12 37](https://user-images.githubusercontent.com/65539896/200058651-085c7b75-c491-44ad-853b-d17da7b618be.jpg)

**Comment by zry98** [comment]
Sí, había una otra persona que tenía esta versión. Tiene un puerto microUSB en el lado derecho y un Tipo-C un poco más arriba.  Adjunto mi suposición de cableado para el microUSB, deberías conectar el pin de OTG ID (junto al pin de GND) al GND (cable negro en el dibujo).  No estoy seguro de si funcionará o no, ¡dinos si lo lograste!

![D37BD0FC-74FF-430F-AF5C-348F5AC9B876](https://user-images.githubusercontent.com/9990159/200088820-d87cc2e7-30a6-49d9-95fd-bd74c3201c2e.jpeg)


![DA6FC973-F839-442B-9587-55B3CBB9ACBA](https://user-images.githubusercontent.com/9990159/200088218-a3b75d7f-e165-43b5-893e-bc8808e0792c.jpeg)

**Comment by Thecloudff7** [comment]
muchas gracis!
voy a probarlo aunque las soldaduras son muy pequeñas  y no tengo mucha experiencia.

voy a mirar un usb hembra en alguna tienda, ya que tenia un hembra recortado de un cable, pero no logro saber que cable va a cada pin

**Comment by iscle** [comment]
Justo ayer compré uno de estos en Wallapop (15 euros) y es un Rev 5. A esta placa le faltan algunos componentes que hace que no funcione el USB. Estoy investigando a ver si descubro qué componentes son y como obtener un puerto funcional. Esto pasa tanto para el USB-C como para el micro USB.

PD: Para el micro USB se tiene que desmontar la placa para ver los componentes que faltan, estan por la parte de abajo.
